cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -113,8 +113,73 @@ func main() {
 		os.Exit(1)
 	}
 
+	// We will try to infer whether the user has an old-style config, or a new
+	// style. Start with the command line argument. If it's true, we know it's
+	// old config style.
+	var oldConfigStyle *bool
```

Comment:
I'd probably keep this as a `bool` rather than `*bool`, so it's easier to work with? Not given it a go, but I think that'd be possible

Reply:
I made it a bool to differentiate between known false and unset. It's hard to know in fallbacks whether something is actually false or zero-value false. Didn't feel like making an optional just for this.